pkg/aks/index.ts

```diff
@@ -0,0 +1,15 @@
+import { importTypes } from '@rancher/auto-import';
+import { IPlugin } from '@shell/core/types';
+import { AKSProvisioner } from './provisioner';
+
+// Init the package
+export default function(plugin: IPlugin): void {
```

Comment:
RE Icon shown in the dashboard's extension list, looks like it also needs config here to show

```
  // Built-in icon
  plugin.metadata.icon = require('./icon.svg');
```

That gives the following 

![image](https://github.com/rancher/dashboard/assets/18697775/3fc7d6a7-12fd-4d12-a6d8-e970d7219763)